pyproject.toml

```diff
@@ -11,7 +11,7 @@ license = "MIT"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
+python = "^3.8"
```

Comment:
Will this allow 3.12?

Reply:
(in other words, why not `>=3.8,<4.0.0`)

Reply:
Yes it will allow 3.12.x. This means exactly that `<=3.8, <4.0.0`